.github/workflows/docs.yml

```diff
@@ -0,0 +1,34 @@
+name: docs
+on:
+  push:
+    branches:    
+      - '**'
+  pull_request:
+    types: [opened, edited]
+jobs:
+  build_docs:
+    name: Build documentation
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Build the Docker image
+        run: |
+          docker build -t ilc-mkdocs - < ./.mkdocs/Dockerfile
+
+      - name: Generate docs and publish them to Netlify
+        run: docker run --rm -v ${PWD}:/docs ilc-mkdocs build
+
+      - name: Deploy to Netlify
+        env:
+          NETLIFY_SITE_ID: 2417a054-8449-4430-8a80-bd1e080d1ad4
+          NETLIFY_AUTH_TOKEN: lAzz9QziMqwZDPfGkpHyr3bXYZsfpVzCzIi_a25FNa4
```

Comment:
credentials has to be moved to github actions credentials

Reply:
Sure, this is a test branch, just to ensure that everything works as expected. The production-ready version will have both creds as Action secrets